standalone/hosts/icmr/playground

```diff
@@ -39,7 +39,13 @@ prometheus
 grafana
 
 [prometheus]
-139.59.4.187
+139.59.87.89
 
 [grafana]
-139.59.4.187
+139.59.87.89
+
+[load_balancing]
+139.59.76.237
```

Comment:
_Technically_ this should be included only in the other PR? Let's just make sure that whichever PR is merged second, takes care of the hosts and the DO box count correctly. 👍

Reply:
Yes, this _shouldn't_ be a part of this PR, although it doesn't make a difference and will need to be fixed when we merge anyways (whichever goes later will conflict).